### ISSUE TITLE
Translation of \mathop{...} to HTML should put some space after the operator

### ISSUE THREAD

**Issue by nrnrnr** [jgm/pandoc#3394]
When $x \notin \mathop{dom} \xi$ is rendered in LaTeX, there is a space between "dom" and the xi.  In HTML there is no such space.  But a space would be appropriate.

Reproducible by

````
echo '$x \notin \mathop{dom} \xi$' | pandoc -t html
````

pandoc --version reports:

````
pandoc 1.19.1
Compiled with pandoc-types 1.17.0.5, texmath 0.9, highlighting-kate 0.6.2.1
Default user data directory: /home/nr/.pandoc
Copyright (C) 2006-2016 John MacFarlane
Web:  http://pandoc.org
This is free software; see the source for copying conditions.
There is no warranty, not even for merchantability or fitness
for a particular purpose.
````

**Comment by jgm** [comment]
texmath currently supports `\operatorname` but not `\mathop`.

http://tex.stackexchange.com/questions/84302/what-is-the-difference-of-mathop-operatorname-and-declaremathoperator

However, even with `\operatorname` it doesn't seem to include a space when rendering to unicode.  That may be an adjustment needed in jgm/texmath.

**Comment by jgm** [comment]
OK, if you compile against the just released texmath 0.9.1, this should now be fixed.
